### PR TITLE
alias, cp, mkdir, mv, touch: add Indonesian translation

### PR DESCRIPTION
**Pull request by widyakumara**
- [x] The page is Indonesian translation of the command `alias`
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages.id/common/alias.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+> Membuat alias -- kata-kata yang digantikan oleh utasan perintah (command)
+> Alias menjadi kadaluarsa sampai sesi shell saat ini berakhir, kecuali jika didefinisikan di file konfigurasi shell, misalnya `~/.bashrc`.
+
+- Menampilkan daftar semua alias:
+
+`alias`
+
+- Membuat alias generik:
+
+`alias {{word}}="{{command}}"`
```
Comment (navarroaxel) [inline pages.id/common/alias.md:12]: Please translate the arguments for get this easier for non-English speakers.

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @widyakumara, welcome and thanks for your submission. I left 1 comment.

Remember that for translations you can send more than 1 translated page in 1 PR, just keep 1 commit per file and no more than 15 pages. This doesn't apply to the primary «English» pages.

File: pages.id/common/alias.md (hunk 1 above)
Comment (widyakumara) [reply pages.id/common/alias.md]: fixed.
also added a few pages per suggestion.

File: pages.id/common/cp.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+`cp {{jalan/menuju/file_sumber.ext}} {{jalan/menuju/file_tujuan.ext}}`
+
+- Menyalin file ke direktori lain, dengan nama yang sama:
+
+`cp {{jalan/menuju/file_sumber.ext}} {{jalan/menuju/direktori_tujuan}}`
+
+- Menyalin sebuah direktori secara beserta isinya ke lokasi lain (jika tujuan sudah ada, direktori tersebut disalin ke dalamnya):
+
+`cp -R {{jalan/menuju/source_directory}} {{jalan/menuju/direktori_tujuan}}`
```
Comment (navarroaxel) [inline pages.id/common/cp.md:15]: Why `source_directory` is not translated? I don't know the language, but I think this is the translation:
```suggestion
`cp -R {{jalan/menuju/direktori_sumber}} {{jalan/menuju/direktori_tujuan}}`
```

File: pages.id/common/touch.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+> Mengubah waktu akses (atime) dan waktu modifikasi (mtime) dari sebuah file.
+
+- Membuat file baru yang kosong atau mengubah waktu file yang telahj ada ke waktu sekarang:
+
+`touch {{nama file}}`
+
+- Mengatur waktu sebuah file ke tanggal dan jam tertentu:
+
+`touch -t {{YYYYMMDDHHMM.SS}} {{nama file}}`
```
Comment (navarroaxel) [inline pages.id/common/touch.md:11]:
```suggestion
`touch -t {{YYYYMMDDHHMM.SS}} {{nama_file}}`
```

File: pages.id/common/touch.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# touch
+
+> Mengubah waktu akses (atime) dan waktu modifikasi (mtime) dari sebuah file.
+
+- Membuat file baru yang kosong atau mengubah waktu file yang telahj ada ke waktu sekarang:
+
+`touch {{nama file}}`
```
Comment (navarroaxel) [inline pages.id/common/touch.md:7]: use `_` for variable names:
```suggestion
`touch {{nama_file}}`
```

File: pages.id/common/touch.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+- Mengatur waktu sebuah file ke tanggal dan jam tertentu:
+
+`touch -t {{YYYYMMDDHHMM.SS}} {{nama file}}`
+
+- Menggunakan waktu dari satu file untuk mengatur waktu file yang lain:
+
+`touch -r {{nama file}} {{nama file 2}}`
```
Comment (navarroaxel) [inline pages.id/common/touch.md:15]:
```suggestion
`touch -r {{nama_file}} {{nama_file2}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
@widyakumara I have a few comments.

File: pages.id/common/touch.md (hunk 5 above)
Comment (widyakumara) [reply pages.id/common/touch.md]: fixed per suggestions

File: pages.id/common/touch.md (hunk 4 above)
Comment (widyakumara) [reply pages.id/common/touch.md]: fixed per suggestions

File: pages.id/common/touch.md (hunk 3 above)
Comment (widyakumara) [reply pages.id/common/touch.md]: fixed per suggestions

File: pages.id/common/mv.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
+# mv
+
+> Memindah atau menamai-ulang file dan direktori
+
+- Memindahkan file ke lokasi yang baru:
+
+`mv {{sumber}} {{target}}`
```
[the remark this answers is not in the document]
Comment (widyakumara) [reply pages.id/common/mv.md]: fixed per suggestions

File: pages.id/common/cp.md (hunk 2 above)
Comment (widyakumara) [reply pages.id/common/cp.md]: fixed per suggestions

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Comment by sbrl** [comment]
Ooops! I did not mean to squash that. I meant to rebase :-/

**Pull request merged**